src/lib/pdfconverter.cc

```diff
@@ -46,7 +46,7 @@ using namespace wkhtmltopdf::settings;
 #define STRINGIZE_(x) #x
 #define STRINGIZE(x) STRINGIZE_(x)
 
-const qreal PdfConverter::millimeterToPointMultiplier = 2.83464567;
```

Comment:
This makes much more sense, as I wasn't sure how this was calculated earlier. So it was basically `72/25.4` (as the default DPI was 72 in WebKit). Does this work when you just simply specify `3.779527559` here (as that's the new default DPI?)

Reply:
Yes it work's fine when I specify `3.779527559`

Reply:
So can you make just that change?

Reply:
This would mean that if WebKit change it's default dpi again the problem will come back ?

Reply:
It won't, it was changed at our end.

Reply:
Ok, I did the change